src/chi/components/carousel/carousel.scss

```diff
@@ -0,0 +1,71 @@
+@import '_global-variables';
+@import '_global-mixins';
+
+.chi-carousel {
+  overflow-x: hidden;
```

Comment:
Looks like the box shadow on the previous and next buttons is getting cut off due to the overflow hidden on chi-carousel. Can we place the overflow hidden on a container that's a sibling to the previous and next buttons instead of a parent?

Reply:
Good catch, done 👍